exercises/wordy/wordy_spec.rb

```diff
@@ -0,0 +1,136 @@
+require_relative "./wordy"
+
+RSpec.describe WordProblem do
+  it "can add one to one" do
+    problem = WordProblem.new("What is 1 plus 1?")
+
+    expect(problem.answer).to eq(2)
+  end
+
+  it "can add two to fifty three" do
+    skip "fix this test second - delete the `skip` lines to enable individual tests"
```

Comment:
`pending` is pretty confusing in Rspec, I think we should stick with `skip`. `pending` actually runs the example and reports errors/failures. That noise is confusing because despite big ugly stack traces, `pending` examples won't fail the run. Quite the opposite, actually. `pending` examples actually fail the run if a `pending` scenario passes.